core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -427,6 +435,10 @@ object ConsumerGroupCommand {
     val describeOpt = parser.accepts("describe", DescribeDoc)
     val deleteOpt = parser.accepts("delete", DeleteDoc)
     val newConsumerOpt = parser.accepts("new-consumer", NewConsumerDoc)
+    val commandConfigOpt = parser.accepts("command-config", CommandConfigDoc)
```

Comment:
It's going to be weird to have both commandConfigOpt and configOpt. Could we just use command-config for both?